### ISSUE TITLE
Staking page/Validator Details

### ISSUE THREAD

**Issue by kanniec826** [forbole/forbole-x-mobile#102]

**Comment by Moistbobo** [comment]
@kanniec826 will the hyperlink be opened with an in-app browser?

**Comment by kanniec826** [comment]
@Moistbobo Yes, in app browser. But thinking can we make something like in ig, there are three dots on the top right hand corner for user to copy link / open in browser in case they want to do that. 

![image.png](https://images.zenhubusercontent.com/610cd72d94b503197fd1700a/77b1c319-644c-4a15-acc5-0b9d19d35fe1)

**Comment by Moistbobo** [comment]
Ok, in that case, it would be good to split the context menu into a separate story.

**Comment by kanniec826** [comment]
#107 created
